deployment/terraform/iam.tf

```diff
@@ -99,6 +102,12 @@ resource "aws_iam_role" "container_instance_ec2" {
   assume_role_policy = "${data.aws_iam_policy_document.container_instance_ec2_assume_role.json}"
 }
 
+resource "aws_iam_role_policy" "ec2_ses_send_email" {
```

Comment:
This one is definitely necessary because it gets assumed by the EC2 instance running the application.